Core/include/Acts/Visualization/ObjVisualization.hpp

```diff
@@ -0,0 +1,215 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2019 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Visualization/IVisualization.hpp"
+
+#include <array>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace Acts {
+
+/// This helper produces output in the OBJ format. Note that colors are not
+/// supported in this implementation.
+///
+template <typename T = double>
+class ObjVisualization : public IVisualization {
```

Comment:
Might be worth reconsidering if having this agnostic to data type is worth having all the code in the header. (Again I know I did this initially.)

Reply:
No problem, can move it into a `src` directory, as I anyway seem to open one up.

Reply:
Done.

Reply:
Exactly, saw that now, but I've put it into an `.ipp` to keep the header clear.